adoc/deployment-preparation.adoc

```diff
@@ -8,42 +8,35 @@ and must be maintained on this machine, but it is not a member of the skuba clus
 [[ssh.configuration]]
 === Basic SSH Key Configuration
 
+*In order to successfully deploy {productname}, you need to have an SSH agent.* This is important, because it is required in order to use the installation tools `skuba` and `terraform`.
+
 [NOTE]
 ====
 The use of `ssh-agent` comes with some implications for security that you should take into consideration.
 
 link:http://rabexc.org/posts/pitfalls-of-ssh-agents[The pitfalls of using ssh-agent]
 
 To avoid these risks please make sure to either use `ssh-agent -t <TIMEOUT>` and specify a time
-after which the agent will self-terminate or terminate the agent yourself before logging out by running
-`ssh-agent -k`.
+after which the agent will self-terminate or terminate the agent yourself before logging out by running `ssh-agent -k`.
```

Comment:
```suggestion
after which the agent will self-terminate or terminate the agent yourself before logging out by running `ssh-agent -k`.,
```
My bad - I left out a comma which I think needs to be in there to indicate where the clauses are separated.